index.html

```diff
@@ -181,17 +181,51 @@ <h2><span lang="en">Stack</span> Technique</h2>
          alt="L'accessibilité se repose avant tout sur le HTML, puis le CSS, le Javascript et enfin ARIA">
 
     <h2>Comment vérifier ?</h2>
+    <img src="outils.webp"
+         alt="">
+    <p>Une partie des tests d'accessibilité est automatisable. Il existe pour cela de nombreux outils comme
+        <a href="https://wave.webaim.org/extension/" target="_blank" rel="noreferrer">WAVE</a>,
+        <a href="https://www.deque.com/axe/" target="_blank" rel="noreferrer">aXe</a>,
+        <a href="https://www.tanaguru.com/" target="_blank" rel="noreferrer">Tanaguru</a> ou
+        <a href="https://asqatasun.org/" target="_blank" rel="noreferrer">Asqatasun</a>. Mettre en place des tests auto est le point de départ.
+    </p>
+    <img src="mains.webp"
+         alt="">
+    <p>La majorité des tests d’accessibilité sont fonctionnels.
+        Ils s’effectuent par des manipulations manuelles. Certains ne nécessitent pas de compétences techniques et
```

Comment:
```suggestion
        Ils s’effectuent par une inspection manuelle. Certains ne nécessitent pas de compétences techniques et
```suggestion
        Ils s’effectuent par des manipulations manuelles. Certains ne nécessitent pas de compétences techniques et
```
```